docs/Security-Compliance/Security-Controls/ac-2-3/index.md

```diff
@@ -0,0 +1,14 @@
+# Access Control
+## AC-02(03) - DISABLE INACTIVE ACCOUNTS  
+
+The information system automatically disables inactive accounts after [a maximum of 180 days of inactivity].  
```

Comment:
It looks like the description from OCIO's Appendix X states that this is a maximum of 60 days, do you know where the 180 day value came from?

Reply:
This came from our conversations around how often some users would log in given the quarterly reporting nature.

Reply:
@lfrohlich that makes sense, I think that should be captured in our own implementation statement rather than changing the language of the control description provided by OCIO though (which states 90 days).

Reply:
Ah yes, definitely

Reply:
@hursey013 @lfrohlich I have updated the file so the description provided by OCIO remains the same, but also added a line to the TDP Implementation portion to state our requirement